### ISSUE TITLE
Preferences/ Crawl web source code (no limits on the number of visits) [Help request]

### ISSUE THREAD

**Issue by fabiosun** [ic005k/OCAuxiliaryTools#234]
Hi, in which condition is useful to flag the preferences option in the thread title?

using that option OCAT has always this error:
<img width="277" alt="Screenshot 2022-05-11 at 8 48 22 AM" src="https://user-images.githubusercontent.com/6358373/167787070-b2c91aee-552f-4c28-98d8-b872eb935c54.png">

option flagged here:
<img width="421" alt="Screenshot 2022-05-11 at 8 48 52 AM" src="https://user-images.githubusercontent.com/6358373/167787117-51a1064a-945d-4203-85a3-df3df5f0d883.png">
additional informations here (in Italian):

[link](https://www.macos86.it/topic/5316-ocat-discussione-e-approfondimenti/?do=findComment&comment=128593)

**Comment by ic005k** [comment]
Oh, it's a very simple question and very easy to choose, if you understand the Github access mechanism.
1. Access through Github API, it will have a limit on the number of times per hour, for more detailed reasons, you can Google the Github API access limit.
2. It is recommended to use crawl web source code, which is not any access restrictions, especially if you access Github without any restrictions, if you access Github with restrictions, it does not matter, you can choose a different mirror server to test the access speed.
In short, you can try different access methods and just find the fastest one that best suits your network situation.

**Comment by ic005k** [comment]
This feature has been redesigned to be more efficient and faster. Please upgrade to the latest version for testing, any questions or suggestions are welcome.
